TabStrip: Docs for Visible and PersistContent parameters. Refactoring the current docs (#619)
telerik/blazor-docs: components/tabstrip/overview.md

This article provides information about the <a href = "https://www.telerik.com/blazor-ui/tabstrip" target = "_blank">Blazor TabStrip component</a> and its core features.

>caption In this article:
* [Basics](#basics)
* [Features](#features)
* [Example](#example)

## Basics

To use a Telerik TabStrip for Blazor:

1. Use the `<TelerikTabStrip>` tag

> 1. Inside it add an instance of `<TabStripTab>` tag for each tab you want to include in the component

Reviewer: ```suggestion
1. Add a nested `<TabStripTab>` tag for each tab you want to include in the component
```

1. Place the desired content in the tabs - it can be any content, including other components

> 1. (Optional) Configure the separate tab instances and their position - Read more in [Tabs Configuration]({%slug tabstrip-tabs-configuration%}) and [Tabs Position]({%slug tabstrip-tabs-position%}) articles.

Reviewer: This point is not easily actionable and does not provide a quick suggestion about a specific setting. I suggest that we directly instruct the reader to set `Title` - before the step about the tab content.

Author: Updated.

![](images/tabstrip-left.png)

## Features

* `Class` - the CSS class that will be rendered on the main wrapping element of the component.
* `ActiveTabIndex` - allows you get and set the currently shown tab index through two-way binding, and also provides an event for the tab change. To deactivate all tabs, set it to `-1`.
* `TabPosition` - allows you to set the desired position of the tab headers. Read more in [Tabs - Position]({%slug tabstrip-tabs-position%}) article.

> * `PersistTabContent` - defines whether the tab content will be preserved if the tab is inactive. Read more in the [Persist Content]({%slug tabstrip-persist-content%}) article.

Reviewer: What is the default value?

```suggestion
* `PersistTabContent` - defines whether the tab content will removed from the DOM when the tab is inactive. Read more in the [Persist Content]({%slug tabstrip-persist-content%}) article.
```
